qiita_pet/templates/edit_study.html

```diff
@@ -157,6 +157,7 @@
 
 {% block content %}
 <h1>{% if study %}Edit study (ID: {{study.id}}) {% else %}Create a new Study{%end%}</h1>
+<h2>If you are planning to load a public study from SRA or EBI, we might be able to help you, please send us the study or project accession to <a href='mailto:qiita.help@gmail.com'>qiita.help@gmail.com</a>.</h2>
```

Comment:
Decided to add this so people starts sending us request ... let's see if it works.